Formula/c/cortexso.rb

```diff
@@ -0,0 +1,29 @@
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://github.com/janhq/cortex/releases/download/v0.4.18/cortex-0.4.18-arm64-mac.tar.gz"
```

Comment:
This looks like a precompiled binary, which is not allowed in Homebrew/core

Reply:
Thank you, we will rework on this